src/aPSFs.jl

```diff
@@ -10,6 +10,53 @@ function apsf(::Type{MethodParaxial}, sz::NTuple, pp::PSFParams; sampling=nothin
     res
 end
 
+function apsf(::Type{MethodCZT}, sz::NTuple, pp::PSFParams; sampling=nothing, center_kz=false) 
```

Comment:
Also: As far as I remember the CZT method propagates different chunks of Z-depths with different propagators. The code I see here seems to only use one such propagator. Also the window size that is used seems to be only 1.2 times the window that would anyway be used. But depending on the propagation depth, this window needs to be bigger to avoid wrap-around problems, I would guess. Can you please send me a picture of the XZ cut of a high-NA (1.4, lambda=500nm) PSF at Nyquist sampling, propagated over 10µm.?